tdvt/tdvt/config_gen/test_creator.py

```diff
@@ -161,3 +162,44 @@ def _process_line(line: str, test_args_dict: List[Dict[str, List]]) -> Tuple[str
             for k in user_col_test_col_map.keys():
                 line = line.replace(k, user_col_test_col_map[k])
             return line + '\n', ignored_line
+
+    def create_custom_expression_tests_for_renamed_staples_and_calcs_tables(self):
+        with open(self.json_file, 'r') as f:
```

Comment:
@rosswbrown and @pvanderknyff what do you think of using a user-provided json object of table mappings? The alternative was using the applicable bits of the logical config. Any preference? This seemed a bit more straight forward.